web-common/src/features/dashboards/time-series/MetricsTimeSeriesCharts.svelte

```diff
@@ -207,6 +209,10 @@
   const setAllMeasuresVisible = () => {
     showHideMeasures.setAllToVisible();
   };
+
+  $: colors = dimensionData.map((d) => {
+    return colorGetter.get($page.params.name, d.dimension, d.value ?? "");
+  });
```

Comment:
Since this is only used for `MeasureChart`, we can move this inside the component.

Reply:
It's placed here because the colors are not unique to each MeasureChart, but rather to the MetricsTimeSeriesChart view.  They only need to be fetched once regardless of the number of charts. Also, because of some upstream issues with TimeSeriesChartContainer and GraphicContext, MeasureChart will re-render dozens of times when you scrub or add new filter pills.